dev-libs/sway/sway-0.6.ebuild

```diff
@@ -0,0 +1,65 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit cmake-utils
+
+DESCRIPTION="i3-compatible Wayland window manager."
+HOMEPAGE="http://swaywm.org/"
+
+SRC_URI="https://github.com/SirCmpwn/sway/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~x86 ~amd64"
+IUSE="X swaylock swaybg swaybar swaygrab swaymsg gdk-pixbuf zsh-completion wallpaper imagemagick ffmpeg systemd"
+
+REQUIRED_USE="swaygrab? ( || ( imagemagick ffmpeg ) )"
```

Comment:
I don't think this is the best way of expressing this. It will commonly cause people to hit blockers and have to disable one of them explicitly. Maybe it would be better to default to the one preferred upstream, and do it like USE=xxx -> use xxx (preferred), USE=-xxx -> use the other.